static/app/components/featureFeedback/feedbackModal.tsx

```diff
@@ -0,0 +1,154 @@
+import {Fragment, useMemo, useState} from 'react';
+import {css} from '@emotion/react';
+import {BrowserClient} from '@sentry/react';
+
+import {addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import Textarea from 'sentry/components/forms/controls/textarea';
+import SelectField from 'sentry/components/forms/selectField';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import OrganizationStore from 'sentry/stores/organizationStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {defined} from 'sentry/utils';
+import {useLocation} from 'sentry/utils/useLocation';
+import useProjects from 'sentry/utils/useProjects';
+
+import ButtonBar from '../buttonBar';
+import Field from '../forms/field';
+
+const feedbackClient = new BrowserClient({
+  // feedback project under Sentry organization
+  dsn: 'https://3c5ef4e344a04a0694d187a1272e96de@o1.ingest.sentry.io/6356259',
```

Comment:
This will also be used for single tenant and self-hosted configurations. Is that ok?

Reply:
I was thinking about the same but was not sure... do we want to allow users to send feedback to another project? but isn't it something that would be relevant only for sentry?

Reply:
I would also see it as a way to opt-out of this feature for self-hosted/single-tenant if needed. But we could also provide a different flag for that. It all depends if we 100% want this hardcoded for all instances?

Reply:
@mitsuhiko do you have any thoughts on this?

Reply:
Alright, we want it everywhere. I reached out to the legal team and have confirmation that it's ok as long as we display this message https://github.com/getsentry/sentry/pull/33858/commits/c6b0d3197e6395a18c8d71947c1ec0f0bb69b30c for self-hosted.